src/message/fetchActions.js

```diff
@@ -222,6 +222,9 @@ export const doInitialFetch = () => async (dispatch: Dispatch, getState: GetStat
   dispatch(initialFetchStart());
   const auth = getAuth(getState());
 
+  dispatch(reportPresence());
+  setInterval(() => dispatch(reportPresence()), 60 * 1000);
```

Comment:
These should go above the `initialFetchStart` and `const auth` lines. Those are closely tied to the `tryUntilSuccessful ... api.registerForEvents` call.